packages/frontend/src/hooks/useQueryResults.tsx

```diff
@@ -48,13 +47,8 @@ export const useQueryResults = (pristine = true) => {
     return useQuery<ApiQueryResults, ApiError>({
         queryKey,
         queryFn: () => getQueryResults(projectUuid, tableId || '', metricQuery),
-        enabled: false, // don't run automatically
```

Comment:
Won't setting enable:true make it automatically fetch each time a component mounts, I am not sure if that behaviour would work well for costly queries.

Reply:
Since the **queryKey** is the same **useQuery** won't make duplicate calls.

Reply:
Actually, I might need to use `refetchOnMount: false` since the default is `always`